### PR TITLE
attempt to fix CircleCI builds

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,8 +30,11 @@ jobs:
           echo $repo
           mkdir -p $repo
           cp -a ./ $repo/
-      - run: |
-          brew test-bot --tap=coin-or-tools/coinor
+      - run: brew install-bundler-gems
+      - run: brew test-bot --only-cleanup-before
+      - run: brew test-bot --only-setup
+      - run: brew test-bot --only-tap-syntax
+      - run: brew test-bot --only-formulae --tap $GITHUB_REPOSITORY
```
Comment (dpo) [inline .circleci/config.yml:37]:
```suggestion
      - run: |
            env
            brew test-bot --only-formulae --tap $GITHUB_REPOSITORY
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,19 +2,20 @@ version: 2
...
     macos:
-      xcode: "11.3.0"
+      xcode: "12.4.0"
     environment:
       CIRCLE_REPOSITORY_URL: https://github.com/coin-or-tools/homebrew-coinor
       HOMEBREW_DEVELOPER: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
       HOMEBREW_FORCE_VENDOR_RUBY: 1
+      GITHUB_REF: refs/pull/$CIRCLE_PR_NUMBER/merge
+      GITHUB_REPOSITORY: coin-or-tools/coinor
```
Comment (dpo) [inline .circleci/config.yml:12]:
```suggestion
      GITHUB_REPOSITORY: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
```

**Comment by dpo** [comment]
@tkralphs I suggest we abandon CircleCI here. Clearly, Homebrew is no longer designed to work with anything else than Github Actions.

**Comment by tkralphs** [comment]
Fine with me!

**Comment by dpo** [comment]
I'm removing it in #68.

**Pull request closed without merging**